app/api/account.py

```diff
@@ -34,7 +34,9 @@ async def add_virtual_lab_account(
     repos: RepoGroupDep, account: VlabAccountCreationIn
 ) -> ApiResponse[VlabAccountCreationOut]:
     """Add a new virtual lab account."""
-    result = await account_service.add_virtual_lab(repos, account_id=account.id, name=account.name)
+    result = await account_service.add_virtual_lab(
+        repos, account_id=account.id, name=account.name, balance=account.balance
```

Comment:
Do we want allow to create a virtual-lab with a balance different from 0?
Wouldn't it create a discrepancy with the double entry accounting?

Reply:
That is extremely good point, Gianluca, thank you for noticing that.
I've updated the service logic to perform an action in two steps: creating an account and after making an top-up transaction on top of that.